src/actions/account.js

```diff
@@ -156,9 +156,12 @@ export const login = ({ passphrase, publicKey, hwInfo }) => async (dispatch, get
     toast.error(getConnectionErrorMessage(error));
     dispatch(accountLoggedOut());
   } else {
+    const loginType = hwInfo
+      ? ['trezor', 'ledger'].find(item => hwInfo.deviceModel.toLowerCase().indexOf(item) > -1)
```

Comment:
`indexOf()` would only match the first index in both the strings so in case we have more complicated strings, this would break but for now it should be fine.

Reply:
All devices that we support have the producer name in their `deviceModel`s, I'm just checking the producer.